### ISSUE TITLE
Uninstaller left over files

### ISSUE THREAD

**Issue by ValooFX** [bradleyasu/ytdl50#1]
Some part of your downloader-installer installs this 'UpLink' downloader executable - as a game which is strange, but also fails (or doesn't want) to remove it when uninstalling. This also seems to be unnecessarily run at startup. Maybe you should check/change the installer system (haven't tested it enough to confirm its harmlessness)

**Comment by bradleyasu** [comment]
Well, uplink.exe is the updater process for the application.  When you say it runs on startup, do you mean startup of your computer or startup of the application?  It should not be running on startup of your computer, I don't know how this could happen.  It should however, run on startup of the application.  It runs in the background to do a quick ping of my update server (http://update.hexotic.net/) to see if a new version is available.

You also mention that it doesn't seem to uninstall when you uninstall the application.  Where are you seeing this?  uplink.exe is installed to "%appdata%\YouTube Downloader 5.0\execs\*" ... is this the location where you are seeing the executable not being deleted?

**Comment by ValooFX** [comment]
Yes, that's the folder that it is left in. As for running on startup, after rebooting and trying to delete it, it was locked as in use by rundll32 (and it also reappeared in the list of recently used items, even though i removed it from there before shutting down), but maybe that was just a coincidence or it's linked to it being registered as a game, or it's a java-thing (no experience with that, just a guess).

**Comment by bradleyasu** [comment]
This is interesting - I've been trying to investigate/replicate this problem and haven't had any luck.  So far this is the only known instance of this happening.   

I do not understand how this could be happening as it's only execute from within the downloader and shouldn't be executed outside by rundll32 or anything else...  I'm also not sure how it could be registered as a game.

**Comment by dietervanhoof** [comment]
I haven't tried any of the above but by just reading this reminded me of Avast Antivirus software that automatically marks certain exe files as games. It doesn't for me but perhaps, depending on which software you run, it might mark it and/or do other stuff with it.

Off-topic: Great tool btw! Thank you for this.
